### PR TITLE
clean up log config

### PR DESCRIPTION
**Pull request by CodiePP**
clean up logging configuration:
metrics should not show up in the logs anymore.

### CODE REVIEW

File: scripts/shelley-testnet.sh [hunk 1]
```diff
@@ -13,8 +13,6 @@ set -e
 # CMD="stack exec --nix --"
 CMD="cabal new-run --"
 
-. $(dirname $0)/lib-node.sh
-
```
Comment (deepfire) [inline scripts/shelley-testnet.sh]: @CodiePP, I think that change should break the script -- this lib defines `nodeargs`, which is used below.

File: configuration/log-config-0.yaml [hunk 2]
```diff
@@ -62,23 +62,27 @@ options:
       # Change the `subtrace` value to `Neutral` in order to log
       # `UTxO`-related messages during epoch validation.
       subtrace: NoTrace
-    '#messagecounters.aggregation':
+    'cardano.#messagecounters.aggregation':
```
Comment (deepfire) [inline configuration/log-config-0.yaml]: Just a stray thought, not really critical -- do we really need this `cardano` prefix everywhere?

It seems that everything in IOHK is about Cardano, somehow, so it seems.. implicit? : -)

Comment (CodiePP) [reply configuration/log-config-0.yaml]: at the moment, yes. there will be changes in the framework on the context naming ... (to be expected)

File: scripts/shelley-testnet.sh (hunk 1 above)
Comment (CodiePP) [reply scripts/shelley-testnet.sh]: it's included twice. one time would be enough?

Comment (deepfire) [reply scripts/shelley-testnet.sh]: Oh, right, my bad! : -)

**Pull request merged**